src/libphonenumber/PhoneNumber.php

```diff
@@ -33,50 +56,101 @@ public function clearCountryCode()
     }
 
     /**
-     * @var int
+     * The national number.
+     *
+     * @var string|null
```

Comment:
I would even suggest to typecast the value in setters to the expected value, to ensure consistency. I can open another PR for this, if you wish.